local_requirements.txt

```diff
@@ -5,6 +5,15 @@ djangorestframework>=3.1,<3.2
 django-ipware==1.1.0
 pytz>=2012h
 pycrypto>=2.6
+python-dateutil==2.1
 
+# edX packages
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
--e git+https://github.com/edx/opaque-keys.git@27dc382ea587483b1e3889a3d19cbd90b9023a06#egg=opaque-keys
+edx-opaque-keys==0.3.4
+
+# Used for testing
```

Comment:
Why do we need this here as well as in `test_requirements.txt`? Maybe we just need to update the documentation to say that those requirements need to be installed as well for local testing.